doc/acl/Everflow-test-plan.md

```diff
@@ -0,0 +1,595 @@
+                                },
+                                "transport": {
+                                    "config": {
+                                        "source-port": "4672..4681"
+                                    }
+                                }
+                            },
+                            "9": {
+                                "actions": {
+                                    "config": {
+                                        "forwarding-action": "ACCEPT"
+                                    }
+                                },
+                                "config": {
+                                    "sequence-id": 9
+                                },
+                                "transport": {
+                                    "config": {
+                                        "destination-port": "4672..4681"
+                                    }
+                                }
+                            },
+                            "10": {
+                                "actions": {
+                                    "config": {
+                                        "forwarding-action": "ACCEPT"
+                                    }
+                                },
+                                "config": {
+                                    "sequence-id": 10
+                                },
+                                "ip": {
+                                    "config": {
+                                        "dscp": "51"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+## PTF Test
+
+### Input files for PTF test
+
+PTF test will generate traffic between ports and make sure it mirrored according to the configured Everflow session and ACL rules. Depending on the testbed topology and the existing configuration (e.g. ECMP, LAGS, etc) packets may arrive to different ports. Therefore ports connection information will be generated from the minigraph and supplied to the PTF script.
+
+### Traffic validation in PTF
+Depending on the test PTF test will verify the packet arrived or dropped.
+
+## Test cases
+
+Each test case will be additionally validated by the loganalizer utility.
+
+Each test case will add dynamic Everflow ACL rules at the beginning and remove them at the end.
+
+Each test case will run traffic for persistent and dynamic Everflow ACL rules.
+
+Each test case will analyze Everflow packet header and payload (if mirrored packet is equal to original).
+
+### Test case \#1 - Resolved route
+
+#### Test objective
+
+Verify that session with resolved route has active state.
+
+#### Test steps
+
+- Create route that matches session destination IP with unresolved next hop.
+- Resolve route next hop.
+- Verify that session state in APP DB changed to active.
```

Comment:
1) the state shall be in state database 2) this step could be eliminated since we have already had tests cover this in virtual switch tests
checking the database adds complexity in this test suite